server/models/DomainEvent.js

```diff
@@ -1,13 +1,65 @@
 const { v4: uuid } = require('uuid');
 const { Repository } = require('./Repository');
 
+const RawRepo = (

```

Comment:
We could rename `RawRepo` function to `DomainEvent` and rename the file to domain-event.js (instead of the current DomainEvent.js).
Generally, I am thinking all files that follow functional model to follow lower case files names with `-` separating two words. For files with classes we could continue to use camelcase file names.